pkgs/applications/science/astronomy/siril/default.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv,
+  fetchgit,
+  pkg-config,
+  meson,
+  ninja,
+  cmake,
+  git,
+  gtk3,
+  wrapGAppsHook,
+  fftwFloat,
+  cfitsio,
+  gsl,
+  exiv2,
+  opencv,
+  libconfig,
+  libraw,
+  libtiff,
+  libpng,
+  libjpeg,
+  ffmpeg,
+  gnuplot,
+  curl
+}:
+
+
+stdenv.mkDerivation rec {
+  pname = "siril";
+  version = "0.99.6";
+
+  src = fetchgit {
+    url = "https://gitlab.com/free-astro/${pname}";
+    rev = "refs/tags/${version}";
+    sha256 = "0prfj6lkzn959gz649hl1kbzpkn509zk3c2x6lrp6kpdssgb3nzy";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ meson ninja git pkg-config cmake ];
+  buildInputs = [
+    gtk3
+    wrapGAppsHook
```

Comment:
this is a nativeBuildInput

Reply:
Moved wrapGAppsHook to nativeBuildInputs in latest commit. Thanks for the review.